PostalCodes.sln

```diff
@@ -13,12 +13,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{E55142
 		.nuget\packages.config = .nuget\packages.config
 	EndProjectSection
 EndProject
-Project("{7CF6DF6D-3B04-46F8-A40B-537D21BCA0B4}") = "PostalCodes.Documentation", "src\PostalCodes.Documentation\PostalCodes.Documentation.shfbproj", "{F4BAE2A2-656D-4413-B09E-091406584EA3}"
```

Comment:
Removing documentation project?

Reply:
Yeah... I was using VS on Mac and did a "shortcut" to fix the build, will revert this.